rm-datasource/src/main/java/io/seata/rm/datasource/exec/InsertExecutor.java

```diff
@@ -209,7 +214,7 @@ protected boolean containsPK() {
     /**
      * check pk values
      * @param pkValues
-     * @return 0 ok 1 exists null value and normal value
+     * @return 0 ok 1 exists null value and non-null value
      */
     private int checkPkValues(List<Object> pkValues) {
```

Comment:
1.why not return boolean
2.unsupport SqlMethodExpr size>1

Reply:
fix done